### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by gavinmdouglas** [NKI-CCB/DISCOVER#15]
Hi there,

I'm getting this error when trying to install the R package with the below command

```
options(repos=c(getOption("repos"), "http://ccb.nki.nl/software/discover/repos/r"))
install.packages("discover")
```

This error is produced:
```
Warning: unable to access index for repository http://ccb.nki.nl/software/discover/repos/r/src/contrib:
  cannot open URL 'http://ccb.nki.nl/software/discover/repos/r/src/contrib/PACKAGES'
```

Thanks,

Gavin

**Comment by scanisius** [comment]
I can't reproduce this issue on my side, which makes solving it a bit more difficult. Could you tell me what operating system and which version of R you are using?

Some things you might try:
- Does changing http:// to https:// in the repository URL fix it?
- Can you access http://ccb.nki.nl/software/discover/repos/r/src/contrib/PACKAGES with your browser?

**Comment by gavinmdouglas** [comment]
Operating system: Ubuntu 20.04.2 LTS
R version 4.1.1

I can access that link on my local computer, but on my server when I try to wget that page I get this error:

```
ERROR: cannot verify ccb.nki.nl's certificate, issued by ‘CN=GEANT OV RSA CA 4,O=GEANT Vereniging,C=NL’:
  Unable to locally verify the issuer's authority.
To connect to ccb.nki.nl insecurely, use `--no-check-certificate'.
```

So maybe this has to do with security settings on the server I'm using that has an issue with your website for some reason. I have been able to download from many other websites on this server with no problem though.

Note that I ran into a similar error about not being able to access the repository with the conda command to install the Python package as well.

No emergency to solve this for my case though as I was able to install the R package from source.


Cheers,

Gavin
